styles/base.css

```diff
@@ -11,11 +11,32 @@ button:disabled {
     opacity: .2;
 }
 
+list row,
+treeview.view,
 window {
     background-color: #13181C;
     -gtk-icon-shadow: none;
 }
 
+treeview.view check {
```

Comment:
this should be with switch

Reply:
no, check is right, because treeview doesnt have renderer for switch, so i tried to solve it by theming check to looks like switch :-/